node.gyp

```diff
@@ -57,6 +57,18 @@
       '<@(node_builtin_shareable_builtins)',
     ],
     'node_sources': [
+      'src/cppnv/EnvKey.cc',
```

Comment:
Can you move all of your implementation to single file, preferably node_dotenv

Reply:
This should be updated in the current push and I reverted your spacing. Once I finish all the requests I will run clang-formatter but for now..